sentry-uitest/sentry-uitest-android-benchmark/src/androidTest/java/io/sentry/uitest/android/benchmark/SentryBenchmarkTest.kt

```diff
@@ -0,0 +1,135 @@
+package io.sentry.uitest.android.benchmark
+
+import android.content.Context
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.core.app.launchActivity
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.espresso.action.ViewActions.swipeUp
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.runner.AndroidJUnitRunner
+import io.sentry.ITransaction
+import io.sentry.Sentry
+import io.sentry.SentryOptions
+import io.sentry.android.core.SentryAndroid
+import io.sentry.uitest.android.benchmark.util.BenchmarkOperation
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.assertTrue
+
+@RunWith(AndroidJUnit4::class)
+class SentryBenchmarkTest {
+
+    private lateinit var runner: AndroidJUnitRunner
+    private lateinit var context: Context
+
+    @BeforeTest
+    fun setUp() {
+        runner = InstrumentationRegistry.getInstrumentation() as AndroidJUnitRunner
+        context = ApplicationProvider.getApplicationContext()
+        context.cacheDir.deleteRecursively()
+        IdlingRegistry.getInstance().register(BenchmarkActivity.scrollingIdlingResource)
+    }
+
+    @AfterTest
+    fun cleanup() {
+        IdlingRegistry.getInstance().unregister(BenchmarkActivity.scrollingIdlingResource)
+    }
+
+    @Test
+    fun benchmarkSameOperation() {
+
+        // We compare two operation that are the same. We expect the increases to be negligible, as the results
+        // should be very similar.
+        val op1 = BenchmarkOperation(runner, getOperation(runner) { null })
+        val op2 = BenchmarkOperation(runner, getOperation(runner) { null })
+        val comparisonResult = BenchmarkOperation.compare(op1, "Op1", op2, "Op2")
+
+        assertTrue { comparisonResult.durationIncrease >= -1 }
+        assertTrue { comparisonResult.durationIncrease < 1 }
```

Comment:
```suggestion
        assertTrue { comparisonResult.durationIncrease in -1..1 }
```
I think it's more readable and also for other assertions